updatersync/src/main/java/nl/openweb/hippo/groovy/watch/FileSystemPoller.java

```diff
@@ -45,16 +47,22 @@
     private final FileAlterationMonitor monitor;
     private FileSystemListener listenerForCurrentChange;
 
-    public FileSystemPoller(final FileFilter fileNameFilter, final long pollingDelayMillis) throws Exception {
+    public FileSystemPoller(final FileFilter fileNameFilter, final long pollingDelayMillis) {
         this.fileNameFilter = fileNameFilter;
         listeners = new HashMap<>();
         monitor = new FileAlterationMonitor(pollingDelayMillis);
-        monitor.start();
+        try {
+            monitor.start();
+        } catch (Exception e) {
+            throw new GroovyFileException("Failed to start filemonitor", e);
+        }
     }
 
     @Override
     public synchronized void registerDirectory(final Path directory, final FileSystemListener listener) throws IOException {
-        log.debug("Registering " + directory);
+        if(log.isDebugEnabled()) {
+            log.debug("Registering {}", directory);
```

Comment:
The if statement should already be inside this log line, duplication of if statement outside the method of log.debug

Reply:
This change was made to fix a sonar code smell.

Reply:
Fair enough.